notNeededPackages.json

```
@@ -3858,6 +3858,10 @@
             "libraryName": "sentence-case",
             "asOfVersion": "1.1.3"
         },
+        "sentry__webpack-plugin": {
+            "libraryName": "@sentry/webpack-plugin",
+            "asOfVersion": "1.10.0"
```

This was added on commit https://github.com/getsentry/sentry-webpack-plugin/commit/b6b0e11e74fd55836d3299cef88531b2a34c2514#diff-7aa4473ede4abd9ec099e87fec67fd57afafaf39e05d493ab4533acc38547eb8 which is part of v1.9.0
1.9.0 changelog: https://github.com/getsentry/sentry-webpack-plugin/releases/tag/v1.9.0
```suggestion
            "asOfVersion": "1.9.0"
```

---

Not possible, there is already a `@types/sentry__webpack-plugin@1.9.3` published on `npm`. I simply took the next version.

---

This is supposed to be the number of the version that added type declarations for this package, it's not the version of the `@types` package

---

The checks won't pass. You can try it out yourself with npm run test-all. This is actually supposed to be the version of the @types package that will be published with the obsoleted warning.

---

Yeah you're right, my bad